cves/CVE-2008-0456.yml

```diff
@@ -115,11 +115,11 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
-  automated:
-  google:
-  contest:
+  answer: This was first reported in 2008, so I wasn't able to find any mailing list that shows where it was actually reported. It looks like RedHat was one vendor that keeps track of these things and they put out a statement saying that this isn't an issue, but other than that there aren't any mailing lists to look at.
```

Comment:
Same note about using pipe to wrap text.

Reply:
Also, minor nitpick: Try to be consistent with spelling of Red Hat. Above you have "Redhat" and here you have "RedHat".